docs/mappers/1_lottery.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 1
+title: Mapper lottery
+---
+
+# Mapper lottery
+To obtain a mapper users can participate in a lottery by buying a ticket with an 
```

Comment:
Too much info in the introduction, this is repeated later and that's a bater place of this kind of details.